clang/runtime/dpct-rt/include/util.hpp.inc

```diff
@@ -283,6 +283,28 @@ vectorized_isgreater<sycl::ushort2, unsigned>(unsigned a, unsigned b) {
 }
 // DPCT_LABEL_END
 
+// DPCT_LABEL_BEGIN|vectorized_isgreater_uchar4|dpct
```

Comment:
for line 279/280:  please check if need update to:  v4[0] = v2[0] > v3[0]?0xff: 0; 
```